token-allocation/src/config.ts

```diff
@@ -0,0 +1,56 @@
+import types from '@chainlink/types'
+import { util } from '@chainlink/ea-bootstrap'
+import { Index } from './adapter'
+import cryptocompare from './data-providers/cryptocompare'
+import nomics from './data-providers/nomics'
+import coinpaprika from './data-providers/coinpaprika'
+import coinmarketcap from './data-providers/coinmarketcap'
+import coingecko from './data-providers/coingecko'
+import coinapi from './data-providers/coinapi'
+import amberdata from './data-providers/amberdata'
+import kaiko from './data-providers/kaiko'
```

Comment:
Do we have no way of using the existing adapters instead of re-writing them here?

Reply:
- When we migrate the price adapters to `.ts`, `token-allocation` will be using them. Right now we can't
- Everything under `data-providers/` is exactly the same on `defi-pulse`, but this will use `token-allocation` adapter after this is merged, so all the repeated code will be removed from `defi-pulse`
- `Synthx` could use `token-allocation` as well, so the repeated code will be removed there too.

Reply:
Thanks for the explanation! Just worried about the technical debt.

Do we have a story for this migration? I feel like we should also include some work on the CVI adapter (which isn't even merged in yet, but I think it can use the same work once it's in).

Reply:
I included 2 stories yes (#176484089 and #176464064). You can include the story for CVI too so we don't forget